jsk_naoqi_robot/README.md

```diff
@@ -34,23 +48,34 @@ wstool merge -t src https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/mast
 wstool update -t src
 source /opt/ros/indigo/setup.bash
 rosdep install -y -r --from-paths src --ignore-src
-catkin build
-source devel/setup.bash
 ```
-% In addition, please install ```ros-indigo-jsk-tools``` to use ```rossetip``` command.
-Otherwise, please write the line below in your ```.bashrc```.
+
+Then, please install Nao/ Pepper mesh files from deb with manual approval of license.  
+Please change 'indigo' to ros distro name which you use.  
+
+```
+sudo apt-get install ros-indigo-pepper-meshes
+sudo apt-get install ros-indigo-nao-meshes
+```
+
+Finally, please compile them.  
 
 ```
-export ROS_IP="133.11.216.yyy"
+catkin build
```

Comment:
`catkin build -c` is better? because many packages are abandoned when one package failed to be compiled. 
https://gist.github.com/kochigami/c29acd6ca562752d834996c6b51f89ad